scripts/circle-publish-npm

```diff
@@ -23,7 +23,7 @@ fi
 if [[ "$branch" == "next" ]]; then
     echo "Publishing with next tag because branch name is next"
     # must set public access for scoped packages
-    pnpm publish --publish-branch "$branch" --tag next --access public
+    pnpm publish --publish-branch next --tag next --access public
```

Comment:
Ah, I guess we still need `--publish-branch` here because the `pnpm-workspace.yaml` setting can't be a list. 😞